docs/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 ### New Features
 
+- Implemented the `execute_buy` and `execute_sell` use cases (invalid inputs are not yet handled)
+- Implemented MarketObserver to update user data when stock market changes
```

Comment:
This line is already included in 0.2.1, you may need to delete this.

Reply:
Already deleted.